packages/web-react/src/components/UNSTABLE_Header/README.md

```diff
@@ -0,0 +1,177 @@
+# UNSTABLE Header
+
+> ⚠️ This component is UNSTABLE. It may significantly change at any point in the future.
+> Please use it with caution.
+
+The `UNSTABLE_Header` component is planned to replace the `Header` component in the future.
```

Comment:
**suggestion (if-minor):** In the next major release, IMHO.

Reply:
Also, could we please deprecate the current `Header` component? Or should we do this in another issue? @crishpeen

Reply:
I think we can deprecate it when we are able to replicate the whole header using the unstable one. Right now, we don't have a Drawer and maybe some other components. I think when we have the full header example in UNSTABLE_Header, we can deprecate the stable one.

Reply:
Ok, I have made an issue for this https://jira.almacareer.tech/browse/DS-1598